.gitlint

```diff
@@ -0,0 +1,54 @@
+[general]
+ignore=title-trailing-punctuation, body-is-missing, body-min-length, title-must-not-contain-word
+
+# ZT: These are custom rules written for Zulip-Terminal, included by default
+extra-path=tools/gitlint-extra-rules.py
+
+# verbosity should be a value between 1 and 3, the commandline -v flags take precedence over this
+# verbosity = 2
+
+# By default gitlint will ignore merge commits. Set to 'false' to disable.
+# ignore-merge-commits=true
+
+# By default gitlint will ignore fixup commits. Set to 'false' to disable.
+# ignore-fixup-commits=true
+
+# By default gitlint will ignore squash commits. Set to 'false' to disable.
+# ignore-squash-commits=true
+
+# Enable debug mode (prints more output). Disabled by default.
+# debug=true
+
+[title-max-length]
+# ZT: Keep this at 72 (Zulip has 76)
+line-length=72
+
+[body-max-line-length]
+# ZT: This matches the Zulip preference for short body line lengths
+line-length=76
+
+[area-formatting]
+# This is a ZT custom rule; these are excluded from the lower-case area rule
+exclusions=WIP, DEBUG
+
+[ignore-by-title]
```

Comment:
I am getting this error when I run `gitlint install-hook`:
`Config Error: No such rule 'ignore-by-title'`

Reply:
What version of gitlint do you have? I have 0.10.0

Are you using an IDE or something like that, which may add git hooks otherwise?

Reply:
Apparently there are two ways to install gitlint. via `apt` and `pip`. If user has already installed gitlint via `apt` then this error occurs.
Uninstalling apt version and then removing the `.git/hooks/commit-msg` file removes the error.

Reply:
Was this an older version of gitlint, or some other issue, do you know?

Reply:
I am sure but it can be an old version of gitlint.